examples/plotting/file/image_rgba_bis.py

```diff
@@ -0,0 +1,29 @@
+import numpy as np
+import requests
+import shutil
+import pylab
+
+from bokeh.plotting import figure, show, output_file
+import bokeh.util.image
+
+# downloading image
+url_lena_gray = 'http://www.ece.rice.edu/~wakin/images/lena512.bmp'
+response = requests.get(url_lena_gray, stream=True)
+with open('lena_gray.bmp', 'wb') as f:
+    shutil.copyfileobj(response.raw, f)
+del response
+
+img_ = pylab.imread('lena_gray.bmp')
+img = bokeh.util.image.convert_gray_to_bokehrbga(img_)
+
+output_file("image.html", title="image.py example")
```

Comment:
Travis is probably failing because our testing machinery needs the same name for the output_file and the file containing the examples... so change to:

`output_file("image_rgba_bis.html", title="image.py example")`